pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "xnat-tools"
-version = "0.1.0"
+version = "0.1.7"
```

Comment:
This is def a breaking change, is your automation going to catch that?

Reply:
I am hoping. When I merge this branch I need to include `BREAKING CHANGE` in the commit message.